geonetwork/log4j/log4j.xml

```diff
@@ -11,6 +11,13 @@
             <param name="ConversionPattern" value="%d{ISO8601} %-5p [%c] - %m%n" />
         </layout>
     </appender>
+    <appender name="harvesterAppender" class="org.apache.log4j.FileAppender">
+        <param name="Threshold" value="ALL" />
+        <param name="File" value="/tmp/harvester/harvester.log" />
```

Comment:
If  `/tmp/harvester/` dir does not exist at runtime, there's a chance this fails.
Please use `/tmp/geonetwork-harvester.log` instead

Reply:
You're right, much better -> https://github.com/georchestra/datadir/pull/86/commits/5c8ca82030112072353608cd3f94846ec58009ef